client/src/components/Graph/drawPointsRegl.ts

```diff
@@ -18,14 +20,29 @@ interface ReglProps {
   scaleref: number;
   spotDiameterFullres: number;
   isSpatial: boolean;
+  duration: number;
+  startTime: number;
 }
 
 export default function drawPointsRegl(regl: Regl) {
   return regl({
     vert: `
+    float easeCubicInOut(float t) {
```

Comment:
Thanks! I've added comprehensive JSDoc-style documentation explaining how the cubic easing function works, including the mathematical steps and purpose.